src/analysis/convex.lean

```diff
@@ -151,16 +145,14 @@ begin
     intros x hx,
     convert segment_translate a b c x _,
     { exact hx.2.symm },
-    { exact hx.1 }
-  },
+    { exact hx.1 }},
```

Comment:
Isn't mathlib style asking for `} }`, with a space in between? (Also downstairs...)

Reply:
That's a good question. I have seen both in mathlib. It would be good to have a precise reference on this. Let me ask on Zulip.

Reply:
Fixed, here and below.